modules/imgproc/include/opencv2/imgproc.hpp

```diff
@@ -3636,6 +3636,23 @@ channels is derived automatically from src and code.
  */
 CV_EXPORTS_W void cvtColor( InputArray src, OutputArray dst, int code, int dstCn = 0 );
 
+/** @brief Converts an image from dual-mode YUV colorspace to RGB colorspace.
```

Comment:
> from dual-mode YUV colorspace to RGB colorspace

Idea of this function is more generic.

Something like this:
"Converts an image from one color space to another, where source image is stored in two planes".